EnglishTranslation-master/QUEST_UNUSED.tsv

```diff
@@ -3091,8 +3091,8 @@ QUEST_UNUSED_20151102_003090	The village resident asked you to contaminate the f
 QUEST_UNUSED_20151102_003091	You found the meat jerky and used the red water to contaminate it. Return to the villager.
 QUEST_UNUSED_20151102_003092	The village chief's grandson is back from checking up on the ferrets after the totem was destroyed. He seems disapponted. Talk to him.
 QUEST_UNUSED_20151102_003093	Dievdirbys Widas says he carved a statue that radiates good energy based on the totem he made for the demons. Talk to Widas again.
-QUEST_UNUSED_20151102_003094	Set up Widas's Statue close to the ferrets
-QUEST_UNUSED_20151102_003095	Dievdirbys Widas says he carved a statue that radiates good energy based on the totem he made for the demons. Set up Widas's statue close to the ferrets and watch their reaction,
+QUEST_UNUSED_20151102_003094	Set up Widas Statue close to the ferrets
+QUEST_UNUSED_20151102_003095	Dievdirbys Widas says he carved a statue that radiates good energy based on the totem he made for the demons. Set up Widas statue close to the ferrets and watch their reaction,
```

Comment:
Set up Widas' statue close to the ferrets and watch their reaction.

(reaction, -> reaction.)